src/curl_multi.cpp

```diff
@@ -21,6 +21,8 @@
 #include <cstdio>
 #include <cstdlib>
 #include <cerrno>
+#include <future>
+#include <thread>
```

Comment:
These two header files are defined in `curl_multi.h`, so there is no need to include them here.

Reply:
OK, that's fine.